packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -208,6 +212,16 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
         commands.registerCommand({ id: 'workbench.action.openSettings' }, {
             execute: () => commands.executeCommand(CommonCommands.OPEN_PREFERENCES.id)
         });
+        commands.registerCommand({ id: 'workbench.extensions.installExtension' }, {
+            execute: async (vsixUriOrExtensionId: UriComponents | string) => {
+                if (typeof vsixUriOrExtensionId === 'string') {
```

Comment:
Mirroring the logic from https://github.com/microsoft/vscode/blob/33a49a9c6328a4576f1d8692bee70c267cbfe16b/src/vs/workbench/contrib/extensions/browser/extensions.contribution.ts#L204.